The program spent 15635.03332734108 s running 3000 times -> xd_scan 11 .csv
The program spent 36175.6731095314 s running 7000 times -> xd_scan 11 .csv
The program spent 51948.8891313076 s running 10000 times -> xd_scan 11 .csv
The program spent 5166.007353067398 s running 1000 times -> xd_scan 11 .csv
The program spent 4230.411707162857 s running 1000 times -> xd_scan 1 .csv
The program spent 45432.96385383606 s running 10000 times -> xd_scan 11 .csv
The program spent 41495.66423845291 s running 8000 times -> xd_scan 11 .csv
The program spent 36265.269644498825 s running 7000 times -> xd_scan 1 .csv
python3: can't open file 'random_general_scan3': [Errno 2] No such file or directory
python3: can't open file 'random_general_scan3': [Errno 2] No such file or directory
The program spent 51078.85515856743 s running 10000 times -> xd_scan 1 .csv
